tests/unit/test-api.cpp

```diff
@@ -222,6 +223,48 @@ const jerry_api_char_ptr_t magic_string_items[] =
 #undef JERRY_MAGIC_STRING_DEF
 };
 
+static bool foreach (const jerry_api_string_t *name,
+                     const jerry_api_value_t *value, void *user_data)
+{
+  char str_buf_p[128];
+  ssize_t sz = jerry_api_string_to_char_buffer (name, (jerry_api_char_t *)str_buf_p, 128);
+  str_buf_p[sz] = '\0';
+  
+  if (!strncmp (str_buf_p, "alpha", (size_t)sz))
+  {
+    JERRY_ASSERT (value->type == JERRY_API_DATA_TYPE_FLOAT64);
+    JERRY_ASSERT (value->v_float64 == 32);
```

Comment:
These two lines are only valid if the 
`CONFIG_ECMA_NUMBER_TYPE` define equals to `CONFIG_ECMA_NUMBER_FLOAT64` it is also possible to have `CONFIG_ECMA_NUMBER_FLOAT32` values. Please see https://github.com/Samsung/jerryscript/blob/master/jerry-core/jerry.cpp#L160 .